### ISSUE TITLE
right to left orientation of the menu

### ISSUE THREAD

**Issue by yossipr** [Microsoft/vscode#20377]
![vs code](https://cloud.githubusercontent.com/assets/4216213/22817067/eb161760-ef6d-11e6-82d5-a32d22869ea9.jpg)
- VSCode Version: Code 1.9.1 (f9d0c687ff2ea7aabd85fb9a43129117c0ecf519, 2017-02-09T00:26:45.394Z)
- OS Version: Windows_NT ia32 10.0.14393
- Extensions:

the menu bar is on the right although the language is LTR (en)

**Comment by smuuf** [comment]
and why is your *start menu* on the right?

**Comment by yossipr** [comment]
because i am from israel and use hebrew as my main language that's RTL.

**Comment by Tyriar** [comment]
Your Hebrew main language configuration would be the reason for this, the English indicator down the bottom is just the input language, not the system language.

**Comment by yossipr** [comment]
I a software engineer and don't understand your answer. The layout should be according the language of the software (en LTR) and not according to layout of OS. I have many software that aligned correctly in en though my main OS language is RTL Hebrew.

**Comment by Tyriar** [comment]
@dbaeumer do you think this would be a limitation of electron?

**Comment by yossipr** [comment]
I don't think so because electron based on html5. Its just bad implementation of Localization.

**Comment by Tyriar** [comment]
@yossipr Electron implements the file menu at the top, we don't support Hebrew so it's probably Electron acting on the system language.

**Comment by yossipr** [comment]
I have diged a little bit. And found that by changing the he.pak file in the locale folder the menu now is in the correct position

**Comment by dbaeumer** [comment]
Electron in this case respects the system setting which is Hebrew. Since VS Code doesn't ship with a Hebrew translation we fall back to a English user interface. I have not seen any API so far how to control the layout in Electron (switch it back to English). 

@yossipr what kind of changes did you do to the `he.pak` file
